### DISCUSSION TITLE
Welcome to evo-wc Discussions!

### DISCUSSION THREAD

**Discussion by intervalia** [EvolvedWeb/evowc#25] [Announcements]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Be kind
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

> Be aware that some discussions may not be showing. You may find it useful to show all of the discussions and not just the open ones.

**Comment by heybran** [comment]
Hi, Brandon here, a developer based in China, previously worked as a front-end developer for a SaaS company based in Australia, mainly working with vanilla Web Components. I'm actively hunting for new remote front-end developer role. Recently spending most of my time working a vanilla Web Components collections project. Love to see web components community grow and embrace the web standard.
